C/COSMA/bundled/patches/openblas.patch

```diff
@@ -0,0 +1,76 @@
+diff --git before/cosma/src/cosma/blas.cpp after/cosma/src/cosma/blas.cpp
```

Comment:
Maybe I'm missing something, but isn't it easier to just rename the symbols with macros, like https://github.com/JuliaPackaging/Yggdrasil/blob/f11bf95fb657f9b7d7bc85228b9aea63f8bb278c/A/armadillo/build_tarballs.jl#L37-L39 (and here they would be much fewer symbols)?

Reply:
Hm yeah, that would probably be easier.

Reply:
Thanks!